Sources/ToosieSlide/UICollectionViewCarouselLayout.swift

```diff
@@ -34,7 +34,7 @@ open class UICollectionViewCarouselLayout: UICollectionViewFlowLayout {
   
   /// The scale factor to apply to the focused cell's height. Defaults to `1.0`.
   /// The height of the item will be multiplied by this value, so a value lower than 1 will make it smaller, greater than 1 will make it bigger.
-  public var focusedItemHeightScaleFactor: CGFloat = 1 { didSet { invalidateLayout() } }
+  public var focusedItemScaleFactor: CGFloat = 1 { didSet { invalidateLayout() } }
```

Comment:
Update the documentation please since it does not only apply to the height anymore